ios-coding-guidelines.md

```diff
@@ -46,4 +46,14 @@ These guidelines are meant to be used without Interface Builder and teach you ho
 
 ## Xcode
 
-1. **No White-Spaces In Group/Folder Names** Scripts do not like whitespaces. Since we have a lot of automation by scripts, please use `_` as replacement for them.
\ No newline at end of file
+1. **No White-Spaces In Group/Folder Names** Scripts do not like whitespaces. Since we have a lot of automation by scripts, please use `_` as replacement for them.
+
+## Accessibility
+
+1. * Use the accessibility id of a view to make it identifiable for unit tests or voice over announcements. The syntax for those identifiers should be `aid.viewController.view` or if further scoping is needed `aid.viewController.parentView.view`.
```

Comment:
I would like to write it in this style: `aid.<view_controller>.<view>` (camel-case and using `< >` which means non-optionl parameter. Optional params are surrounded by `[ ]`)

For large lists it is better to always write `1.` for the numbers so you can easily place a new item in the middle of them without the need to renumber the whole list.

You don't need the `\*` in front of each item when you have the numbers.

Reply:
Hm what do you mean with "always write `1.`?

Reply:
ah you can write the following and don't care about the numbering in markdown:
```
1. 
1.
1.
1.
1.
```

Reply:
It will be formatted correctly